design/FY2024/NFP-labHeatExchangerHR.md

```diff
@@ -0,0 +1,270 @@
+          \type choice
+          \key Plate
+          \key Rotary
+          \default Plate
+      A9, \field Frost Control Type
+          \type choice
+          \key None
+          \key ExhaustAirRecirculation
+          \key ExhaustOnly
+          \key MinimumExhaustTemperature
+          \default None
+      N7, \field Threshold Temperature
+          \type real
+          \units C
+          \default 1.7
+          \note Supply (outdoor) air inlet temp threshold for exhaust air recirculation and
+          \note exhaust only frost control types. Exhaust air outlet threshold Temperature for
+          \note minimum exhaust temperature frost control type.
+      N8, \field Initial Defrost Time Fraction
+          \type real
+          \units dimensionless
+          \minimum 0.0
+          \maximum 1.0
+          \default 0.083
+          \note Fraction of the time when frost control will be invoked at the threshold temperature.
+          \note This field only used for exhaust air recirc and exhaust-only frost control types.
+      N9, \field Rate of Defrost Time Fraction Increase
+          \type real
+          \units 1/K
+          \minimum 0.0
+          \default 0.012
+          \note Rate of increase in defrost time fraction as actual temp falls below threshold temperature.
+          \note This field only used for exhaust air recirc and exhaust-only frost control types.
+      A10,\field Economizer Lockout
+          \type choice
+          \key Yes
+          \key No
+          \default Yes
+          \note Yes means that the heat exchanger will be locked out (off)
+          \note when the economizer is operating or high humidity control is active
+      A11,\field Sensible Effectiveness of Heating Air Flow Curve Name
+          \note optional
+          \note if this field has value, then the sensivle effectiveness for heating
+          \note will be the value in N2 multiplied this curve value
+          \type object-list
+          \object-list UnivariateFunctions
+      A12,\field Latent Effectiveness of Heating Air Flow Curve Name
+          \note optional
+          \note if this field has value, then the latent effectiveness for heating
+          \note will be the value in N3 multiplied this curve value
+          \type object-list
+          \object-list UnivariateFunctions
+      A13,\field Sensible Effectiveness of Cooling Air Flow Curve Name
+          \note optional
+          \note if this field has value, then the sensivle effectiveness for cooling
+          \note will be the value in N4 multiplied this curve value
+          \type object-list
+          \object-list UnivariateFunctions
+      A14;\field Latent Effectiveness of Cooling Air Flow Curve Name
+          \note optional
+          \note if this field has value, then the latent effectiveness for cooling
+          \note will be the value in N5 multiplied this curve value
+          \type object-list
+          \object-list UnivariateFunctions
```

Comment:
1) Does there really need to be curves for cooling and heating? isn't is just effectiveness versus air flow? and 2) what happens when these curves are not used?, the effectiveness is constant? I don't recall the conference call discussion on this.

Reply:
The notes are updated in the newer commits. The curves are multipliers to the 100% airflow effectiveness value. The multiplier will be a function of airflow. If the curve is not there the value will be 0. There are four curves as sensible and latent effectiveness for both heating and cooling will need to be specified.